react/features/base/lastn/middleware.ts

```diff
@@ -102,15 +78,9 @@ MiddlewareRegistry.register(store => next => action => {
     switch (action.type) {
     case APP_STATE_CHANGED:
     case CONFERENCE_JOINED:
-    case PARTICIPANT_JOINED:
-    case PARTICIPANT_KICKED:
-    case PARTICIPANT_LEFT:
-    case SELECT_LARGE_VIDEO_PARTICIPANT:
     case SET_AUDIO_ONLY:
     case SET_CAR_MODE:
     case SET_FILMSTRIP_ENABLED:
-    case SET_TILE_VIEW:
```

Comment:
Doesn't the tile view need a re-calculation?

Reply:
Nope, I can't think of a case when last-n needs to be recalculated after tile view is toggled. Last-n should only change when 

- audio-only/car mode is toggled
- filmstrip is toggled
- Pip mode is toggled 
- Quality control slider is moved